src/transformers/generation/logits_process.py

```diff
@@ -322,11 +328,18 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
 
 class EncoderRepetitionPenaltyLogitsProcessor(LogitsProcessor):
     r"""
-    [`LogitsProcessor`] enforcing an exponential penalty on tokens that are not in the original input.
+    [`LogitsProcessor`] enforcing an exponential penalty on tokens that are not in the original input. This technique
+    avoids hallucination by boosting the probabilities of tokens found within the original input.
+
+    While the intended usage is to penalize and thus reduce hallucination, this technique can also be used to reward and
+    thus encourage creativity in a similar manner. To penalize and reduce repetition, use `hallucination_penalty` values
+    above 1.0, where a higher value penalizes more strongly. To reward and encourage creativity, use 
+    `hallucination_penalty` values between 0.0 and 1.0, where a lower value rewards more strongly.
 
     Args:
         hallucination_penalty (`float`):
```

Comment:
Proof of my poor past review: this arg should match the one in `__init__`. Would you be so kind as to update `hallucination_penalty` to `penalty` here?

(we can't do it the other way around because of backwards compatibility :) )